### DISCUSSION TITLE
My student authentication application has been approved, and it has been a long time since it was approved, but the benefits have not been issued yet

### DISCUSSION THREAD

**Discussion by Ch3nSir** [community/community#139469] [GitHub Education]
### Hi everyone,

Dear administrator,plz help me,thanks

**Comment by davevad93** [comment]
Hi @Ch3nSir ,

Since when you've been approved? The waiting time is usually a few days to a maximum of a couple of weeks and may vary depending on different factors, such as the volume of requests.

More information can be found in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

You can also take a look at the following discussions for further reference: 
- #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Reply by anwarbasha2002** [reply]
I too didn't received any reply from since I applied I got approved but benefits were not available.
